### ISSUE TITLE
github workflow not supported

### ISSUE THREAD

**Issue by climba03003** [fastify/github-action-merge-dependabot#179]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the regression has not already been reported

### Last working version

3.0.2

### Stopped working in version

3.1.0

### Node.js version

-

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

-

### 💥 Regression Report

I think it should fallback to title regex when no `package.json` is found on diff.
Of cuz, it should not allow to upgrade if the title do not match which is different then before.

### Steps to Reproduce

https://github.com/fastify/fastify/pull/3825

### Expected Behavior

github workflow should be supported.

**Comment by simoneb** [comment]
good catch, caused by not having good enough code coverage which we're addressing now
